web/modules/custom/greet/src/Controller/GreetController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\greet\Controller\GreetController.
+ */
+namespace Drupal\greet\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+class GreetController extends ControllerBase {
+  public function overview() {
+    return array(
+      '#type' => 'markup',
+      '#markup' => 'Hello '. \Drupal::currentUser()->getDisplayName(),
```

Comment:
This is not the expected way to use services in class files.
Check the page on Services and Dependency injection in Drupal docs.